langlib/lang.value/src/main/ballerina/src/lang.value/value.bal

```diff
@@ -228,3 +228,13 @@ public isolated function mergeJson(json j1, json j2) returns json|error = @java:
     'class: "org.ballerinalang.langlib.value.MergeJson",
     name: "mergeJson"
 } external;
+
+# Converts a field in map<json> to a user-specified basic simple type.
+# The basic simple types are ()|int|float|decimal|string|boolean.
+# + v - field value of map<json> or json value
+# + t - basic simple type to convert to
+# + return - value belonging to `t`, or error if this cannot be done
+public isolated function requireType(any|error v, typedesc<any> t) returns any|error =  @java:Method {
```

Comment:
In the proposal, the function name is `ensureType `

https://github.com/ballerina-platform/ballerina-spec/issues/487#issuecomment-674467479

Reply:
Fixed.